iceoryx_examples/waitset/CMakeLists.txt

```diff
@@ -60,6 +60,12 @@ target_link_libraries(iox-ex-waitset-trigger
 )
 target_compile_options(iox-ex-waitset-trigger PRIVATE ${ICEORYX_WARNINGS} ${ICEORYX_SANITIZER_FLAGS})
 
+add_executable(iox-ex-waitset-basic ./ice_waitset_basic.cpp)
+target_link_libraries(iox-ex-waitset-basic
+    iceoryx_posh::iceoryx_posh
+)
+target_compile_options(iox-ex-waitset-basic PRIVATE ${ICEORYX_WARNINGS} ${ICEORYX_SANITIZER_FLAGS})
```

Comment:
Can you please replace `ex` by `cpp`? This is done for all other binary names in #655.

Reply:
Sure. Did not like the ex anyway. Reminds me of an ex you know:P